examples/resources/pingfederate_authentication_policies_settings/import.sh

```diff
@@ -0,0 +1,2 @@
+# "authenticationPoliciesSettingsId" should be "id"
```

Comment:
This is probably best for another issue, and I know this is copied from our other examples, but this comment doesn't seem accurate. In fact with resources like this that authenticationPoliciesSettingsId value shouldn't even matter, no matter what the user supplies.

Reply:
OK, I'll leave it here for consistency and we can do a global fix after researching this out.